glusterd2/commands/volumes/brick-replace-txn.go

```diff
@@ -0,0 +1,98 @@
+package volumecommands
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/brick"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/pkg/api"
+)
+
+func prepareBricks(c transaction.TxnCtx) error {
+	var req api.BrickReq
+	if err := c.Get("newBrick", &req); err != nil {
+		return err
+	}
+	err := PrepareBrick(req, c)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func replaceVolinfo(c transaction.TxnCtx) error {
+	var newBrick api.BrickReq
+	if err := c.Get("newBrick", &newBrick); err != nil {
```

Comment:
create an err variable and use it instead of creating a new variable in each  look